### DISCUSSION TITLE
How can I disable neovide_backtraces.log?

### DISCUSSION THREAD

**Discussion by oriongonza** [neovide/neovide#2053] [General]
My filesystem is completely polluted with `neovide_backtraces.log`!  Is there a way to disable them with a configuration file or something?

**Comment by baltermia** [comment]
Same here. Pretty frustrating!

**Reply by baltermia** [reply]
Opened an issue for this #2145.

**Reply by oriongonza** [reply]
I fixed mine by actually looking at the backtrace and fixing it. It's still annoying though!

**Reply by baltermia** [reply]
ye I should've just looked at the log haha, thought this was some kind of other issue
